topics/dev/images/Makefile

```diff
@@ -0,0 +1,4 @@
+
+all:
+	test -f plantuml.jar ||  wget http://jaist.dl.sourceforge.net/project/plantuml/plantuml.jar
```

Comment:
Would there be any reason to not just include the jar? Would save the downloading every time, and make it more offline-friendly to run?

Reply:
Up to you - seems like overkill to me. This only gets downloaded once per clone essentially since it stores it right in the directory. It is offline ready after the first generation. Putting it in repo would mean everyone would need to download it every clone but currently only one person has ever re-generated the figures. Also it is GPL which... I think would be fine to redistribute since we're not linking against it but is still more dicey than a less copy-left license.

Reply:
we've thought about it a bit more, your decision seems good :) lets stick with that.